src/blocks/menu.scss

```diff
@@ -0,0 +1,66 @@
+.menu {
+  background-color: $colorBlue;
+  box-sizing: border-box;
+  height: 100vh;
+  padding: 24px 0;
```

Comment:
You have padding top here and for menu-content, that's why your menu icon and cross don't locate at the same place